stabilizer.py

```diff
@@ -1252,6 +1253,17 @@ def renderfile(self, starttime, stoptime, outpath = "Stabilized.mp4", out_size =
         if type(viewer_thread) != type(None):
             viewer_thread.map_function_enable = old_map_enable_setting
 
+        audio_codec = "copy"
+
+        try:
```

Comment:
This should only be needed if audio export is enabled. In other words, it should be inside `if audio:` below.